docs/dev/porting_legacy_tests.md

```diff
@@ -0,0 +1,90 @@
+
+From [Shanghai](https://ethereum.org/en/history/#shapella) onward, new test cases — especially for new features introduced in hard forks—are defined in Python within EEST. While the existing test cases remain important for client testing, porting ethereum/tests to EEST will help maintain and generate tests for newer forks. This also ensures feature parity, as client teams will only need to obtain test fixture releases from one source.
+
+While automating the conversion of the remaining YAML (or JSON) test cases to Python is possible, manually porting individual test cases offers several benefits:
+
+- Reducing the number of test cases by combining multiple YAML (or JSON) cases into a single Python test function using parametrization.
+- Potentially improving coverage by parametrizing the Python version.
+- Producing higher quality code and documentation, which are typically clearer than an automated conversion.
+- Ensuring better organization of tests within the `./tests` folder of execution-spec-tests by fork and EIP.
+
+## Porting an original test
+
+1. Select one or more YAML test cases from ethereum/tests to port and create an issue in this repository AND comment on [this tracker issue.](https://github.com/ethereum/execution-spec-tests/issues/972)
+
+2. [Add a new test](../writing_tests/index.md) in the appropriate fork folder, following the guidelines for [choosing a test type.](../writing_tests/types_of_tests.md#deciding-on-a-test-type)
+
+3. Submit a PR with the ported tests:
+
+   1. Add the list of ported YAML files to [`converted-ethereum-tests.txt`](https://github.com/ethereum/execution-spec-tests/blob/1b30c336eae6b0746ea4db441ac74406f2fb2322/converted-ethereum-tests.txt).
+   2. Open a PR to remove the ported tests from the _original tests_ repository.
+
+> See also: 📄 [Getting started with EEST.](../getting_started/repository_overview.md)
+
+## Filling tests
+
+EEST uses pytest to run tests against EELS (an EVM implementation for testing). This process is known as "filling" and verifies the assertions in your tests. You can use the fill CLI for this. For example, see how to fill the `PUSH` opcode.
+
+```shell
+uv run tests/frontier/opcodes/test_push.py
+```
+
+See also: 📄 [Documentation for the `fill` command.](../filling_tests/filling_tests_command_line.md)
+
+> If the tests can't currently be filled, please explain the issue (feel free to also [open a Discussion](https://github.com/ethereum/execution-spec-tests/discussions/new?category=general)).
+
+## Debugging tests
+
+By default, EVM logs are stored in the `logs` folder at the repository root. You can check the `output` folder to review transaction results. If needed, review a previous PR that ported tests (e.g., [the PR porting the `PUSH` opcode](https://github.com/ethereum/execution-spec-tests/pull/975)).
```

Comment:
I just added a new label `port` for ported tests and labelled the ones I could find. Could you add a link to that so people can find related PRs easily?
https://github.com/ethereum/execution-spec-tests/pulls?q=is%3Apr+label%3Aport
But do please leave a link to this as an example (which is not tooo long or complicated).

Reply:
If you think the label should be more descriptive, let me know.